weave-js/src/components/Dialog/Dialog.tsx

```diff
@@ -51,9 +51,11 @@ export const Overlay = React.forwardRef(
  */
 const contentClassName = classNames(
   'night-aware',
-  'rounded border border-moon-250 bg-white py-24 px-32 shadow-md',
-  'dark:border-moon-750 dark:bg-moon-900 dark:text-moon-200',
-  'absolute left-[50%] top-[50%] translate-x-[-50%] translate-y-[-50%]' // centers modal on screen
+  'absolute left-[50%] top-[50%] translate-x-[-50%] translate-y-[-50%]', // centers modal on screen
+  'rounded border py-24 px-32',
+  'shadow-lg shadow-oblivion/[0.16] dark:shadow-oblivion/[0.48]',
+  'border-moon-250 bg-white bg-white text-moon-850 ',
+  'dark:border-moon-750 dark:bg-moon-900 dark:text-moon-150'
```

Comment:
Note: dark mode doesn't have a overlay in storybook because i'm waiting for designers to come back with a value for me to use

![Screenshot 2023-11-17 at 12 43 40 PM](https://github.com/wandb/weave/assets/103448723/d3428ce3-16c4-450f-8816-08832ef69e26)

![Screenshot 2023-11-17 at 12 43 35 PM](https://github.com/wandb/weave/assets/103448723/93323a37-83fd-4673-a1f3-848dd063405e)

Usage in weave in reports:

![Screenshot 2023-11-17 at 12 39 54 PM](https://github.com/wandb/weave/assets/103448723/64e8443e-f3a0-4430-8cf2-8687c8da60a2)


Usage in core app:
![Screenshot 2023-11-17 at 12 40 40 PM](https://github.com/wandb/weave/assets/103448723/b8b7c428-d599-4084-b3e8-32c867af6fad)
![Screenshot 2023-11-17 at 12 41 17 PM](https://github.com/wandb/weave/assets/103448723/7e965aff-f791-4a89-8752-db240d45fa53)
![Screenshot 2023-11-17 at 12 41 00 PM](https://github.com/wandb/weave/assets/103448723/3bf359dc-7776-4216-90bf-e074c5f30009)
![Screenshot 2023-11-17 at 12 41 10 PM](https://github.com/wandb/weave/assets/103448723/0c37f4de-0768-4c38-897a-e98026c1805b)